text/0000-types-team.md

```diff
@@ -0,0 +1,241 @@
+
+## Initial details
+
+Here are some details of the team that are true at the time of this writing. They are expected to change over time.
+
+### Leads and membership
+
+The current team leads are Nicholas Matsakis (nikomatsakis) and Jack Huey (jackh726). Initial membership will be determined by the leads after team is created.
+
+### Team meetings
+
+In general, each active initiative coordinates its own activity. For team-wide, cross-initiative communication, the team currently has a single weekly meeting which serves two purposes:
+
+* The first meeting of the month is the **planning meeting**, in which we review status updates and schedule the topics to cover in the remaining meetings of that month.
+* Subseqent meetings are **deep dive meetings**, in which we spend 60-90 minutes doing a deep read through a PR, design document, or other material and having team-wide discussion.
+
+These meetings are currently held on Zulip.
+
+#### Planning meeting
+
+We review the overall roadmap for each active initiative and set some kind of goals.
+
+Each initiative is responsible for preparing a short (1-2 paragraph) document in the leadup to the planning meeting with the following structure:
+
+* What was the plan at the start of the month (list of goals)?
+* What happened? (brief narrative)
+* What is the plan for this month (list of goals)?
+
+The meeting begins by reading these documents and asking questions. The goal is to adjust the goals for each initiative so that they are realistic; we should be helping each other to calibrate and set expectations. The final document is then published as a blog post.
+
+#### Deep dive meeting
+
+A "deep dive" meeting takes the form of reviewing a write-up, a PR, or otherwise diving into some topic together. They are expected to last 90 minutes but perhaps longer. Deep dive meetings for a given month are scheduled during the planning meeting. 
+
+An [example transcript of a previous deep dive meeting can be found here.](https://zulip-archive.rust-lang.org/stream/144729-wg-traits/topic/deep.20dive.202022-03-18.3A.20intro.20to.20formality.html)
+
+### Github Projects
+
+The types team maintains the following projects:
+
+* [chalk](https://github.com/rust-lang/chalk) and chalk-ty
+* [polonius](https://github.com/rust-lang/polonius)
+* [a-mir-formality](https://github.com/nikomatsakis/a-mir-formality)
+    * this may eventually grow to multiple 'formality' models
+
+The team is also responsible for those portions of [rust-lang/rust] that implement the type system. This will be done jointly, with the compiler team, since those portions are not cleanly separable. Specific details of how this is done, however, are left out of this RFC, as they will likely change over time.
+
+[a-mir-formality]: https://github.com/nikomatsakis/a-mir-formality
+[rust-lang/rust]: https://github.com/rust-lang/rust
+
+### Active initiatives
+
+The currently active initiatives of the trait team are as follows:
```

Comment:
```suggestion
The currently active initiatives of the types team are as follows:
```

Also, does this belong in the initial RFC? If so, should it be framed as being included for the sake of example, since all of these items will become dated in the future, while the types team itself won't?

Reply:
This entire "Initial details" section is meant to be "this is how things are, but not necessarily how they will stay"-- that could perhaps be clarified.

Reply:
Actually we do say:

> Here are some details of the team that are true at the time of this writing. They are expected to change over time.